Change 9f2 from rounding to truncation (#9)
CarterKoala/wca-regulations-wrc: wca-regulations.md

# <wca-title>WCA Regulations

> <version>Version: January 1, 2025

Reviewer: ```suggestion
<version>Version: January 1, 2024
```

        - 9b5a) Full round formats for this event: "Best of X" (where X is 1, 2, or 3).
        - 9b5b) Cutoff formats for this event: "Best of X" (where X is 1 or 2) cutoff phase for "Best of Y" (where Y is 2 or 3, and Y > X).
- 9f) The results of a round are measured as follows:

>     - 9f1) All timed results under 10 minutes, except for 3x3x3 Multi-Blind, are measured and truncated to the nearest hundredth of a second. All timed averages and means under 10 minutes are measured and rounded to the nearest hundredth of a second.

Author: ```suggestion
    - 9f1) All timed results under 10 minutes, except for 3x3x3 Multi-Blind, are measured and truncated to hundredths of a second. All timed averages and means under 10 minutes are measured and rounded to the nearest hundredth of a second.
```

>     - 9f2) All timed results, averages, and means over 10 minutes, as well as all times for 3x3x3 Multi-Blind results, are measured and truncated to the nearest second.

Author: ```suggestion
    - 9f2) All timed results, averages, and means over 10 minutes, as well as all times for 3x3x3 Multi-Blind results, are measured and truncated to seconds.
```
